src/runtime.js

```diff
@@ -417,9 +421,14 @@ export function adopt(global) {
  * @param {!Window} global Global scope to adopt.
  */
 export function adoptShadowMode(global) {
+
+  const doc = global.document,
+    isCriticalPath = !!doc.querySelector('script[shadow-doc-for-critical-path]');
```

Comment:
I'm trying to understand what exactly "critical path" means. Is this simply an option to the doc on the shell?